pyintelowl/cli/tags.py

```diff
@@ -31,6 +32,45 @@ def ls(ctx: ClickContext, as_json: bool):
         ctx.obj.logger.fatal(str(e))
 
 
+@tags.command(help="Create new Tag")
+@click.argument("label", type=str)
+@click.argument("color", type=str)
+@add_options(json_flag_option)
+@click.pass_context
+def new(ctx: ClickContext, as_json: bool, color: str, label: str):
+    ctx.obj.logger.info("Adding new Tag..")
+    try:
+        ans = ctx.obj.create_tag(label, color)
+        if as_json:
+            rprint(json.dumps(ans, indent=4))
+        else:
+            ans = [ans]
+            _print_tags_table(ctx.obj.logger, ans)
+    except IntelOwlClientException as e:
+        print(e.response.status_code)
+        ctx.obj.logger.fatal(str(e))
+
+
+@tags.command(help="Edit existing Tag attributes")
+@click.argument("tag_id", type=int)
+@click.argument("label", type=str)
+@click.argument("color", type=str)
+@add_options(json_flag_option)
+@click.pass_context
+def edit(ctx: ClickContext, as_json: bool, color: str, label: str, tag_id: int):
+    ctx.obj.logger.info("Editing new Tag..")
+    try:
+        ans = ctx.obj.edit_tag(tag_id, label, color)
+        if as_json:
+            rprint(json.dumps(ans, indent=4))
+        else:
+            ans = [ans]
+            _print_tags_table(ctx.obj.logger, ans)
```

Comment:
I would prefer to not mutate `ans` variable here, so on line 68, we can directly write `_print_tags_table(ctx.obj.logger, [ans]). Same for line 47 and 48.
`